frontend/src/pages/workspace/[slug]/1-project-team/setup-team.js

```diff
@@ -1,13 +1,40 @@
+import { useState } from 'react'
 import { PageLayout } from '..'
+import SetupTeamForm from '../../../../modules/workspace/ps/setup-team-form'
+import SetupTeamTable from '../../../../modules/workspace/ps/setup-team-table'
+import styles from './setup-team.module.scss'
 
-const View = () => (
-  <>
-    <h4 className="caps-heading-m">
-      National steering committee & Project Team
-    </h4>
-    <h2 className="h-xxl w-bold">Setup Your Team</h2>
-  </>
-)
+const View = ({ psItem }) => {
+  const [members, setMembers] = useState([])
+  return (
+    <>
+      <div className={styles.titleSection}>
+        <h4 className="caps-heading-m">
+          National steering committee & Project Team
+        </h4>
+        <h2 className="h-xxl w-bold">Create Your Team</h2>
+      </div>
+      <div className={styles.descSection}>
+        <p>
+          Establish a National Steering Committee (NSC) with representation from
+          relevant government agencies as well as relevant non-governmental
+          organisations. The NSC will aim to meet 2-4 times per year to assist
+          with and drive the roadmap/strategy/plan. Establish a project team
+          consisting of consultants to lead the implementation of the project
+          with the guidance of the NSC and UNEP
+        </p>
+      </div>
+      <div className={styles.tableSection}>
```

Comment:
This is not the preferred way to attach styling.
Rather we want to have one root class, for example: `setup-team-view` and then to have nested scss under that instead of attaching every single <div> element...

Reply:
noted, I'll refactor it and follow the preferred pattern